tests/brevitas_finn/brevitas_examples/test_mobilenet_finn_export.py

```diff
@@ -17,7 +17,7 @@
 
 from brevitas import torch_version
 from brevitas.export import export_finn_onnx
-from brevitas_examples.imagenet_classification import quant_mobilenet_v1_4b
```

Comment:
Can we retain the import from imagenet_classification? you just need to import things from the init